mindsdb/integrations/utilities/datasets/question_answering/squad_v2_val_100_sample.csv

```diff
@@ -0,0 +1,110 @@
+id,title,context,question,answers
```

Comment:
this is a 100 row sample for squad v2 dataset, I propose we use this as first benchmark set.

We can ofc add other popular ones going fwds